docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: p0stgr3s_pass
+    ports: 
+      - '5432:5432'
+    volumes:
+      - calendso-db:/var/lib/postgresql/data
+
+  calendso:
+    image: calendso
+    environment:
```

Comment:
An `env.example` file has already been created (and theoretically, a `.env` which will be the copy of this file). Rather than specifying each and every environment variable (and make mistakes when one is added to the `.env.example`) it would be preferable to use the `env_file: .env` property to load all userland environment variable specified in the `.env` file. See https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option